tide_grant.module

```diff
@@ -70,6 +70,25 @@ function tide_grant_config_ignore_settings_alter(array &$settings) {
  */
 function tide_grant_webform_submission_presave($submission) {
   if ($submission->getWebform()->get('id') === 'tide_grant_submission') {
+    $has_sites = FALSE;
+    $node_sites = [];
+    $node_primary_site = 0;
```

Comment:
Because these aren't used unless Tide Site is installed they should be nested in the conditional.

Reply:
Thanks @GROwen , changed.